tools/diff-processor/labels/get_issue.go

```diff
@@ -20,7 +20,7 @@ func GetIssue(repository string, id uint64) (labeler.Issue, error) {
 		return issue, fmt.Errorf("Error creating request: %w", err)
 	}
 	req.Header.Add("Accept", "application/vnd.github+json")
-	req.Header.Add("Authorization", "Bearer "+os.Getenv("GITHUB_TOKEN"))
+	req.Header.Add("Authorization", "Bearer "+os.Getenv("GITHUB_TOKEN_MAGIC_MODULES"))
```

Comment:
Ideally this would also be passed in - but it's nested deeply enough (and not using our github client anyway) that this seems like a reasonable compromise 👍